scripts/processing/01-anatomy.py

```diff
@@ -93,19 +98,29 @@ def process_subject_anat(subject_id, force_recon_all=False):
         print('  Making BEM')
         mne.bem.make_flash_bem(subject, subjects_dir=subjects_dir,
                                show=False, verbose=False)
-    fname_bem_surfaces = op.join(subjects_dir, subject, 'bem',
-                                 '%s-5120-5120-5120-bem.fif' % subject)
-    if not op.isfile(fname_bem_surfaces):
-        print('  Setting up BEM model')
-        bem_model = mne.make_bem_model(
-            subject, ico=4, subjects_dir=subjects_dir)
-        mne.write_bem_surfaces(fname_bem_surfaces, bem_model)
-    fname_bem = op.join(subjects_dir, subject, 'bem',
-                        '%s-5120-5120-5120-bem-sol.fif' % subject)
-    if not op.isfile(fname_bem):
-        print('  Computing BEM solution')
-        bem = mne.make_bem_solution(bem_model)
-        mne.write_bem_solution(fname_bem, bem)
+    for n_layers in (1, 3):
```

Comment:
why do we need to do this?

Reply:
I was trying to get 3-layer BEMs to work. I'm optimistic that it might at some point. In the meantime it shouldn't hurt